### DISCUSSION TITLE
Bug Report: (Inside Build 22567) UWP App Icon dispear in Start Menu

### DISCUSSION THREAD

**Discussion by Shomnipotence** [valinet/ExplorerPatcher#988] [General]
OS Version: **Build 22567**.ni-release.220225-1325
App Version: 22000.493.42.28 Pre-release

Screenshot
![image](https://user-images.githubusercontent.com/6630660/156686715-8d7224e0-74b3-490c-9e15-abfd3945e24f.png)

 UWP App Icon dispear in Start Menu

**Comment by ghost** [comment]
Hi :) 

I had similar issues with UWP apps on the startmenu have you tried restarting a few times plus resetting the explorer patcher to it`s default and switch back to the Windows 10 startmenu

Shawn

**Reply by Shomnipotence** [reply]
No I unistall the patch, everything become normal

**Reply by Shomnipotence** [reply]
Because you cannot change the settings.

**Comment by OrthodoxWindows** [comment]
I have the impression that UWP applications causes much more frequent problems than Win32 applications, in the development of ExplorerPacher and other software.
